sorc/build_all.sh

```diff
@@ -37,8 +56,13 @@ source ./machine-setup.sh > /dev/null 2>&1
 #------------------------------------
 # INCLUDE PARTIAL BUILD 
 #------------------------------------
+. ./partial_build.sh $@
 
-. ./partial_build.sh
+if [ $target = jet ]; then
```

Comment:
Lines 61-65 and lines 67-71 are identical.   Is one of these blocks intended to be used for `$target` not equal to `jet`?

Reply:
Ugh...how does this duplicate block keep getting back in?! @WalterKolczynski-NOAA haven't we already removed this block at least twice before? So odd!

Reply:
Should be fixed now.